pkgs/tools/security/ghidra/extensions/wasm/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGhidraExtension,
+  ghidra,
+  ant,
+}:
+let
+  version = "2.3.1";
+in
+buildGhidraExtension {
+  pname = "wasm";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "nneonneo";
+    repo = "ghidra-wasm-plugin";
+    rev = "v${version}";
+    hash = "sha256-aoSMNzv+TgydiXM4CbvAyu/YsxmdZPvpkZkYEE3C+V4=";
+  };
+
+  # compile sleigh
+  configurePhase = ''
+    runHook preConfigure
+
+    pushd data
+    ${ant}/bin/ant -f build.xml -Dghidra.install.dir=${ghidra}/lib/ghidra sleighCompile
```

```suggestion
    ant -f build.xml -Dghidra.install.dir=${ghidra}/lib/ghidra sleighCompile
```
ant should be added to nativeBuildInputs

---

done :+1: